zuul_alpha/core.py

```diff
@@ -96,17 +106,31 @@ def _set_encryptor(self, public_key):
                     "public key '%s' does not exist." % self.public_key_file)
 
         try:
-            public_key_obj = RSA.import_key(public_key).publickey()
-            self.encryptor = PKCS1_OAEP.new(public_key_obj)
+            public_key_obj = serialization.load_pem_public_key(
```

Comment:
For RSA, yes, I had confirmed by copying a `zuul_data` with test keys and secrets dir produced from master into this branch and it decrypted. However for AES, it is not as Cryptography uses Fernet and Pycryptodomex uses EAX.

We haven't deployed yet so all (large) secrets from the start would be using Fernet AES.